plugins/ua_config_default.c

```diff
@@ -20,6 +20,22 @@
 #include "ua_securitypolicies.h"
 #include "ua_plugin_securitypolicy.h"
 
+#ifndef UA_SEND_BUFFER_SIZE_DEFAULT
+# define UA_SEND_BUFFER_SIZE_DEFAULT 65535 /* 64k per chunk */
+#endif
+
+#ifndef UA_RECV_BUFFER_SIZE_DEFAULT
+# define UA_RECV_BUFFER_SIZE_DEFAULT 65535 /* 64k per chunk */
+#endif
+
+#ifndef UA_MAX_MESSAGE_SIZE_DEFAULT
+# define UA_MAX_MESSAGE_SIZE_DEFAULT 0 /*  0 -> unlimited */
+#endif
+
+#ifndef UA_MAX_CHUNK_COUNT_DEFAULT
+# define UA_MAX_CHUNK_COUNT_DEFAULT 0 /*  0 -> unlimited */
+#endif
```

Comment:
Please do not add these macros. They add needless complexity.
We should keep compile-time flags to a minimum.

If an embedded developer has little ram available, then he can always override the settings once the client has been instantiated (before starting it).
Or by writing a custom function `UA_ClientConfig_setMyEmbeddedDefault`.

Reply:
Ok, I'll do that. Thanks!